server/models.py

```diff
@@ -236,7 +236,7 @@ class Meta:
         unique_together = ("machine", "update")
 
 class Plugin(models.Model):
-    name = models.CharField(max_length=512, unique=True)
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
Django 1.8 performs a check (https://docs.djangoproject.com/en/1.8/ref/checks/#database) to make sure that “max_length” is no greater than 255 when using “unique=True” with MySQL. I do use MySQL so I changed it so I could push it my server with a “git pull.”

Otherwise, you get this error:
	SystemCheckError: System check identified some issues:

	ERRORS:
	server.Plugin.name: (mysql.E001) MySQL does not allow unique CharFields to have a max_length > 255.

Thanks,
Chase Thompson-Baugh
801.866.9174

> On Jul 5, 2015, at 6:49 AM, Graham Gilbert <notifications@github.com> wrote:
> 
> In server/models.py <https://github.com/salopensource/sal/pull/1#discussion_r33894596>:
> 
> > @@ -236,7 +236,7 @@ class Meta:
> >          unique_together = ("machine", "update")
> >
> >  class Plugin(models.Model):
> > -    name = models.CharField(max_length=512, unique=True)
> > +    name = models.CharField(max_length=255, unique=True)
> Why was this change needed?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/salopensource/sal/pull/1/files#r33894596>.
>